docs/src/main/sphinx/release/release-411.md

```diff
@@ -0,0 +1,59 @@
+# Release 411 (15 Mar 2023)
+
+## General
+
+* Add spilled data size to query statistics. ({issue}`16442`)
+* Add {func}`sinh` function. ({issue}`16494`)
+* Add support for a `GRACE PERIOD` clause in the `CREATE MATERIALIZED VIEW`
+  task. This is a backwards incompatible change, and the previous behavior can
```

Comment:
Maybe we should expand more what "backwards incompatible change" means here in practice. This can be done in context of Iceberg connector only though